### PR TITLE
fix: Info Label Platform changes with doc changes

### PR DESCRIPTION
**Pull request by Lokanathan-k**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/2338

#### Please provide a brief summary of this pull request.

This PR is mainly for converting InfoLabel directive from core to Info Label Component in platform
The Info Label component is used to highlight the status of an information by coloring the text.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done

Documentation checklist:
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by fkolar** [comment]
Is this new component ? Why this is FEAT ?

File: libs/platform/src/lib/components/info-label/info-label.component.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit, ElementRef, ViewEncapsulation } from '@angular/core';
+import { InfoLabelComponent, CssClassBuilder, applyCssClass } from '@fundamental-ngx/core';
+
+@Component({
+    selector: 'fdp-info-label',
+    templateUrl: './info-label.component.html',
+    encapsulation: ViewEncapsulation.None
+})
+export class InfoLabelPlatformComponent extends InfoLabelComponent implements CssClassBuilder {
```
Comment (fkolar) [inline libs/platform/src/lib/components/info-label/info-label.component.ts:9]: What is the added value  ? How we are enhancing this component ?

**Comment by Lokanathan-k** [comment]
> Deploy preview for _fundamental-ngx_ ready!
> 
> Built with commit [46ad37e](https://github.com/SAP/fundamental-ngx/commit/46ad37e30e7f2b9a93e9e1208b8ffca808647d0f)
> 
> https://deploy-preview-2407--fundamental-ngx.netlify.app

Since platform does not have infolabel, i have named it as feat, since it is not a fix or not a docs changes i have made it a feat

**Comment by fkolar** [comment]
> > Deploy preview for _fundamental-ngx_ ready!
> > Built with commit [46ad37e](https://github.com/SAP/fundamental-ngx/commit/46ad37e30e7f2b9a93e9e1208b8ffca808647d0f)
> > https://deploy-preview-2407--fundamental-ngx.netlify.app
> 
> Since platform does not have infolabel, i have named it as feat, since it is not a fix or not a docs changes i have made it a feat

Again, So the added value that we are wrapping it with component ?  If yes, then the implementation should be simplier:

- You dont need css builder
- You dont need custome_schema 
- You should be using core in this case not duplicating core unless you have some reason for it. 

@manjunathanagaraj Why we are starting implementation on this ? When we start on new component we need to align on component signature, and have technical spec. I dont see it for this component.

**Comment by Lokanathan-k** [comment]
> > > Deploy preview for _fundamental-ngx_ ready!
> > > Built with commit [46ad37e](https://github.com/SAP/fundamental-ngx/commit/46ad37e30e7f2b9a93e9e1208b8ffca808647d0f)
> > > https://deploy-preview-2407--fundamental-ngx.netlify.app
> > 
> > 
> > Since platform does not have infolabel, i have named it as feat, since it is not a fix or not a docs changes i have made it a feat
> 
> Again, So the added value that we are wrapping it with component ? If yes, then the implementation should be simplier:
> 
> * You dont need css builder
> * You dont need custome_schema
> * You should be using core in this case not duplicating core unless you have some reason for it.
> 
> @manjunathanagaraj Why we are starting implementation on this ? When we start on new component we need to align on component signature, and have technical spec. I dont see it for this component.

**Comment by Lokanathan-k** [comment]
> > > Deploy preview for _fundamental-ngx_ ready!
> > > Built with commit [46ad37e](https://github.com/SAP/fundamental-ngx/commit/46ad37e30e7f2b9a93e9e1208b8ffca808647d0f)
> > > https://deploy-preview-2407--fundamental-ngx.netlify.app
> > 
> > 
> > Since platform does not have infolabel, i have named it as feat, since it is not a fix or not a docs changes i have made it a feat
> 
> Again, So the added value that we are wrapping it with component ? If yes, then the implementation should be simplier:
> 
> * You dont need css builder:- css class are applied twice over the core component and as well as wrapper component.
> * You dont need custome_schema
> * You should be using core in this case not duplicating core unless you have some reason for it.: its not duplicated but just inherited from the core component.
> 
> @manjunathanagaraj Why we are starting implementation on this ? When we start on new component we need to align on component signature, and have technical spec. I dont see it for this component.

link for the technical spec:

this spec was created Using Object status as reference:
https://github.com/SAP/fundamental-ngx/wiki/Platform:-Info-Label-Component-Technical-Design

**Comment by manjunathanagaraj** [comment]
> > > Deploy preview for _fundamental-ngx_ ready!
> > > Built with commit [46ad37e](https://github.com/SAP/fundamental-ngx/commit/46ad37e30e7f2b9a93e9e1208b8ffca808647d0f)
> > > https://deploy-preview-2407--fundamental-ngx.netlify.app
> > 
> > 
> > Since platform does not have infolabel, i have named it as feat, since it is not a fix or not a docs changes i have made it a feat
> 
> Again, So the added value that we are wrapping it with component ? If yes, then the implementation should be simplier:
> 
> * You dont need css builder
> * You dont need custome_schema
> * You should be using core in this case not duplicating core unless you have some reason for it.
> 
> @manjunathanagaraj Why we are starting implementation on this ? When we start on new component we need to align on component signature, and have technical spec. I dont see it for this component.

**Comment by Lokanathan-k** [comment]
> Doesn’t look to have breaking changes so why is It feat

done

File: libs/platform/src/lib/components/info-label/info-label.component.ts (hunk 1 above)
Comment (Lokanathan-k) [reply libs/platform/src/lib/components/info-label/info-label.component.ts:9]: Conversion of directive to Component

File: apps/docs/src/app/platform/component-docs/platform-info-label/platform-info-label-docs.component.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<fd-docs-section-title [id]="'default'" [componentName]="'infolabel'">
```
Comment (salarenko) [inline apps/docs/src/app/platform/component-docs/platform-info-label/platform-info-label-docs.component.html:1]: Using multiple time the same id attribute values

File: apps/docs/src/app/platform/platform-documentation.module.ts [hunk 3]
```diff
@@ -127,7 +133,14 @@ import { ScrollingModule } from '@angular/cdk/scrolling';
         PlatformRadioGroupListExampleComponent,
         PlatformRadioGroupContentExampleComponent,
         PlatformRadioGroupDisabledExampleComponent,
-        PlatformRadioGroupDocsComponent
+        PlatformRadioGroupDocsComponent,
```
Comment (salarenko) [inline apps/docs/src/app/platform/platform-documentation.module.ts:136]: Could you guys consider creating lazy loaded modules?
Pulling all this code at once affects our documentation page usage experience.

Comment (fkolar) [reply apps/docs/src/app/platform/platform-documentation.module.ts:136]: make sense. not only docs but also tests.

**Comment by fkolar** [comment]
@Lokanathan-k I think you should close this PR.

File: apps/docs/src/app/platform/platform-documentation.module.ts (hunk 3 above)
Comment (Lokanathan-k) [reply apps/docs/src/app/platform/platform-documentation.module.ts:148]: @fkolar @salarenko this change will be taken in a diffrent PR

**Comment by Lokanathan-k** [comment]
@salarenko i am facing test issue with the your latest code push can you help me with this issue.

Screenshot 2020-05-07 at 10 40 47 PM

**Pull request closed without merging**